terraform/oracle/cam-ora/oraclient/main.tf

```diff
@@ -0,0 +1,151 @@
+######################################################################
+#  Copyright: IBM Corp., 2019

```

Comment:
Apache2 license?

Reply:
Do you have a "boiler plate" I should use for the header?

Reply:
Updated all tf and code files to show Apache 2.0 license